data-raw/README.md

```diff
@@ -0,0 +1,11 @@
+This directory contains scripts to download and preprocess data used within the rOpenScPCA package.
+There are also notebooks that explore some of the prepared datasets for exploration.
+
+## Building Gene References
+
+- The `build_gene_references.R` script creates a table of Ensembl id to gene symbol references.
+It starts with the example ScPCA-formatted SCE object stored in test data, extracting the table of gene ids and gene symbols.
+This is combined with the reference information extracted from example 10x Genomics datasets.
+The full table is saved in `data/scpca_gene_reference.rda`.
```

Comment:
```suggestion
The script extracts the table of gene ids and gene symbols from the example ScPCA-formatted SCE object and combines it with the reference information extracted from example 10x Genomics datasets.
The full table is saved in `data/scpca_gene_reference.rda`.
```

Reply:
take it or leave it, your text is also fine.

actually though, should we say "example sce" or "test sce"?

Reply:
Might also say that re-running this script will overwrite the data

Reply:
Updated the text